### PR TITLE
Feature/reverse proxy

### PR DESCRIPTION
**Pull request by qvalentin**
tested with:

http://localhost/api/swagger-ui/

http://localhost/api/health

### CODE REVIEW

File: lib/install.sh [hunk 1]
```diff
@@ -65,7 +60,7 @@ ffinstall() {
   if ! [[ $db_password ]]; then
     # Create new Password
     echo "Creating new random password for the database."
-    db_password=$(wget -qO- "https://www.passwordrandom.com/query?command=password&scheme=rrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrr")
+    db_password=$(wget -qO- "https://www.passwordrandom.com/query?command=password&scheme=rrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrr")
```
Comment (open-schnick) [inline lib/install.sh:63]: xD

**Review by open-schnick — COMMENTED** [review]
I am gonna implement the forwarding to swagger-ui

**Comment by qvalentin** [comment]
> I am gonna implement the forwarding to swagger-ui
but /api/swagger-ui/ works, should be good enough

**Comment by open-schnick** [comment]
> > I am gonna implement the forwarding to swagger-ui
> > but /api/swagger-ui/ works, should be good enough

you are right xD

**Comment by qvalentin** [comment]
> > > I am gonna implement the forwarding to swagger-ui
> > > but /api/swagger-ui/ works, should be good enough
> 
> you are right xD

is it so weird. NGIX takes /api/health and maps it to /health  (same with swagger). IDK why the rest api works with this?

**Comment by open-schnick** [comment]
> > > > I am gonna implement the forwarding to swagger-ui
> > > > but /api/swagger-ui/ works, should be good enough
> > 
> > 
> > you are right xD
> 
> is it so weird. NGIX takes /api/health and maps it to /health (same with swagger). IDK why the rest api works with this?

wait what.

**Comment by qvalentin** [comment]
> > > > > I am gonna implement the forwarding to swagger-ui
> > > > > but /api/swagger-ui/ works, should be good enough
> > > 
> > > 
> > > you are right xD
> > 
> > 
> > is it so weird. NGIX takes /api/health and maps it to /health (same with swagger). IDK why the rest api works with this?
> 
> wait what.

From inside the container:

root@4264d0916932:/# curl FileFighterREST:8080/health
{"uptimeInSeconds":1437,"userCount":1}root@4264d0916932:/#

**Comment by qvalentin** [comment]
> > > > > > I am gonna implement the forwarding to swagger-ui
> > > > > > but /api/swagger-ui/ works, should be good enough
> > > > 
> > > > 
> > > > you are right xD
> > > 
> > > 
> > > is it so weird. NGIX takes /api/health and maps it to /health (same with swagger). IDK why the rest api works with this?
> > 
> > 
> > wait what.
> 
> From inside the container:
> 
> root@4264d0916932:/# curl FileFighterREST:8080/health
> {"uptimeInSeconds":1437,"userCount":1}root@4264d0916932:/#

joke

**Pull request merged**